tests/QA/module.tests.ps1

```diff
@@ -1,112 +1,207 @@
-$here = Split-Path -Parent $MyInvocation.MyCommand.Path
-
-# Convert-path required for PS7 or Join-Path fails
-$ProjectPath = "$here\..\.." | Convert-Path
-$ProjectName = (Get-ChildItem $ProjectPath\*\*.psd1 | Where-Object {
-    ($_.Directory.Name -match 'source|src' -or $_.Directory.Name -eq $_.BaseName) -and
-    $(try { Test-ModuleManifest $_.FullName -ErrorAction Stop }catch{$false}) }
-).BaseName
-
-$SourcePath = (Get-ChildItem $ProjectPath\*\*.psd1 | Where-Object {
-        ($_.Directory.Name -match 'source|src' -or $_.Directory.Name -eq $_.BaseName) -and
-        $(try { Test-ModuleManifest $_.FullName -ErrorAction Stop }catch { $false }) }
+BeforeDiscovery {
```

Comment:
Since you fixed this file, we should have it updated in the template as well.
Can be done in separate PR though.

Reply:
Yes, I will do it in another PR. We must discuss if we should support both pester 4 and Pester 5 in the templates, or if we change everything to Pester 5.